lib/ManualImageCrop.php

```diff
@@ -207,6 +207,23 @@ public function cropImage() {
 		$src_w = max(0, $_POST['select']['w']) * $_POST['previewScale'];
 		$src_h = max(0, $_POST['select']['h']) * $_POST['previewScale'];
 		
+		$size = wp_get_image_editor( $src_file )->get_size();
+		
+		$is_higher = ( $dst_h > $size["height"] );
```

Comment:
$size gets original uploaded image dimensions so $is_higher and $is_wider (booleans) are variable condition to check what scale should be done in next steps.